tests/utils.py

```diff
@@ -0,0 +1,91 @@
+import pytest
+from unittest import mock
+import numpy as np
+import pyfar as pf
+from haiopy.buffers import SignalBuffer
+
+
+def default_devices():
+    return [0, 0]
+
+
+def query_devices(id=None, kind=None):
+    if kind == 'input':
+        return {
+            'name': "MockDevice",
+            'index': 0,
+            'hostapi': 'CoreAudio',
+            'max_input_channels': 8,
+            'default_low_input_latency': 0.1,
+            'default_high_input_latency': 0.15,
+            'default_samplerate': 44100
+        }
+    elif kind == 'output':
+        return {
+            'name': "MockInputDevice",
```

Comment:
Why do we use `'name': "MockInputDevice"`, when `kind == 'output'`? Same question for `kind=None` and `kind=input`.